schemas/Batch.yaml

```diff
@@ -1,10 +1,33 @@
 type: object
 required:
-  - skuId
-  - defaultLabTestingState
-  - approvalStatus
-  - approvalStatusDateTime
+  - id
+  - brand_id
+  - coa_ids
+  - updated_on
 properties:
+  id:
+    $ref: Id.yaml
+  brand_id:
```

Comment:
Noticing a lot of snake-case. I would prefer to keep json formatted with camelcase to 
1. match styles 
2. Be consistent with JSON formatted APIs 

I'd like to follow the google JSON guide for formatting these fields; https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format